powersimdata/input/profile_input.py

```diff
@@ -35,13 +38,20 @@ def get_profile_version(_fs, grid_model, kind):
     return [f.replace(f"{kind}_", "").replace(".csv", "") for f in matching]
 
 
+def _make_fs():
+    mfs = MultiFS()
+    writeable = server_setup.BLOB_TOKEN_RW is not None
+    mfs.add_fs("profile_fs", get_blob_fs("profiles"), write=writeable)
+    return mfs
```

Comment:
using `MultiFS` wrapper since there is code in `data_access.py` which relies on this in some cases